scripts/docker-env

```diff
@@ -6,3 +6,9 @@ if [[ "$(docker version -f {{.Server.Version}})" =~ ^17\. ]]; then
     DELEGATED=:delegated
 fi;
 
+# allow scripts to be passed special argument that triggers a docker pull
+# without this images can get stale and out of sync from CI system
+if [ "$1" = "pull" ]; then
+    exec docker pull "${CONTAINER_NAME}"
+    exit $?
```

Comment:
FWIW, when you use `exec` it replaces the current process, making the `exit` an unreachable statement

Reply:
Normally yes, but for things failing (e.g. `exec` not having enough memory to process its work) better have a fallback plan :)